pkg/kubeapiserver/options/authorization.go

```diff
@@ -95,6 +95,8 @@ func (s *BuiltInAuthorizationOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&s.PolicyFile, "authorization-policy-file", s.PolicyFile, ""+
 		"File with authorization policy in json line by line format, used with --authorization-mode=ABAC, on the secure port.")
+	fs.MarkDeprecated("authorization-policy-file", "Authorization mode ABAC is deprecated. "+
+		"Consider using the RBAC or Webhook authorization modes instead. Will be removed in a future version.")
```

Comment:
I'd recommend putting a specific target release here. Since the config file is a beta-level API, that means 9 months or 3 releases (whichever is longer). If this is deprecated in v1.17, that means dropping it in v1.20.

Reply:
Fixed.